src/server/check-answers/controller.js

```diff
@@ -73,6 +74,18 @@ export const checkAnswersPostController = {
       })
     }
 
+    const emailContent = Object.values(tasks)
+      .flatMap(({ questionPageAnswers }) =>
+        questionPageAnswers.map(
+          ({ page, answer }) =>
+            `## ${page.question} \n ${answer.html.replace(/<br \/>/g, '\n')}`
+        )
+      )
+      .join('\n')
+
+    await sendNotification({
```

Comment:
I'm curious to know how this is going to work under test - e.g. in github actions, or in local testing.  Have you had any thoughts about this?